src/graphql/types/object/consumer-account.ts

```diff
@@ -43,10 +45,25 @@ const ConsumerAccount = new GT.Object({
     csvTransactions: {
       type: GT.NonNull(GT.String),
       args: {
-        walletIds: {
-          type: GT.NonNullList(GT.ID),
+        walletNames: {
+          type: GT.NonNullList(WalletName),
         },
       },
+      resolve: async (source, args) => {
+        const walletIds: WalletId[] = []
+
+        for (const id of source.walletIds) {
```

Comment:
This looks like duplication - should be calling a use-case method not live in the resolver.

Reply:
Done